src/main/app/claims/claimStoreClient.ts

```diff
@@ -171,4 +171,36 @@ export class ClaimStoreClient {
       .get(`${claimStoreApiUrl}/${reference}/defendant-link-status`)
       .then(linkStatus => linkStatus.linked)
   }
+
+  retrieveUserRoles (user: User): Promise<String> {
+    if (!user) {
+      return Promise.reject(new Error('User must be set'))
+    }
+
+    return this.request
+      .get(`${claimApiBaseUrl}/user/roles`, {
+        headers: {
+          Authorization: `Bearer ${user.bearerToken}`
+        }
+      })
+      .then((roles: string[]) => {
+        return roles.join(',')
+      })
+  }
+
+  persistRoleName (user: User, roleName: string): Promise<String> {
```

Comment:
How about:
`addRoleToUser()`

and could you add a little background please about why this is here? That it is a tactical solution and the data will be stored in idam eventually